include/flashinfer/gemm/fp4_gemm_template_sm100.h

```diff
@@ -273,7 +273,7 @@ size_t genericFp4GemmKernelLauncher(void* D, void const* A, void const* B, void
                            std::string(cutlassGetStatusString(initStatus));                                  \
       throw std::runtime_error("[FP4 gemm Runner] " + errMsg);                                               \
     }                                                                                                        \
-    auto runStatus = gemm.run(args, workspace, stream, nullptr, /* enablePDL */ false);                      \
```

Comment:
@yzh119 may we make it configurable, for some use cases, I don't want to enable it by default